src/python3/Makefile

```diff
@@ -15,6 +15,7 @@ DERIVED_TARGETS = lib$(PYTHON_PNAME)-stdlib_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd6
 				  #$(PYTHON_PNAME)-dev_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -fr $(PYTHON_PNAME)-$(PYTHON_VER)
```

Comment:
>rm -fr $(PYTHON_PNAME)-$(PYTHON_VER) [](start = 1, length = 36)

This fix could be separated into standalone PR and complete quickly.

Reply:
ok